docs/source/library/python/concrete_drivers/index.rst

```diff
@@ -0,0 +1,23 @@
+:mod:`concrete-drivers` --- Concrete Sensor Driver
+==================================================
+
+The classes in these modules are the concrete drivers that are composed from
+the base :py:class:`ale.base.base.Driver` class and various mix-ins classes.
```

Comment:
I wanted to add a reference to the mix-in index here, but I couldn't get it to work for some reason.